README.md

```diff
@@ -2,33 +2,60 @@
 
 Theme for using GitBook to publish an FAQ or Knowledge Base. This theme works perfectly with search plugins (as the [default one](https://github.com/GitbookIO/plugin-search) or [algolia](https://github.com/GitbookIO/plugin-algolia)).
 
-### Usage
+## Usage
 
 This theme requires GitBook version 3 or later.
 
 Add the theme to your book's configuration (`book.json`):
 
 ```js
 {
-    "plugins": ["theme-faq"]
+    "plugins": [
+        "theme-faq",
+        "-fontsettngs",
+        "-sharing"
+        ]
 }
 ```
 
-##### Add relations between articles
+**NOTE** `theme-faq` is not compatible with plugins that modify the toolbar (since there is no toolbar). Embedded search will not work as a result. 
 
-Suggestions for other articles can be shown at the bottom of an article, relations are specified in the YAML frontmatter of a page:
+This includes the default plugins `fontsettings` and `sharing` that need to be disabled explicitly (add a minus flag "-" before each plugin parameter).
 
+If you disable both, the search bar will work. Otherwise the javascript fails when loading the page.
```

Comment:
Mentionning the incompatible plugins was totally needed. I'll just strip this last sentence about JavaScript since it is superfluous.